backend/src/main/scala/com/softwaremill/adopttapir/infrastructure/Json.scala

```diff
@@ -1,12 +1,20 @@
 package com.softwaremill.adopttapir.infrastructure
 
 import com.softwaremill.tagging.@@
-import io.circe.generic.AutoDerivation
 import io.circe.{Decoder, Encoder, Printer}
+import io.circe.generic.extras.{AutoDerivation, Configuration => CirceConfiguration}
 
 /** Import the members of this object when doing JSON serialisation or deserialisation.
   */
 object Json extends AutoDerivation {
+
+  private val DiscriminatorName = "type"
```

Comment:
the whole point in extracting this as a val was to define this in a single place and share among `Json`/`Http` :)

Reply:
Moved `DiscriminatorName ` to `Constants`